woocommerce/woocommerce
# [CYS on core] Introduce the Transitional page, i2

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Implement the new transitional page for the Customize Your Store flow, including the following updates:

- Updated subheading copy.
- Updated copy for the "View store" button
- Remove the store preview frame.
- Updated the Actions section with a new "Add your products" item to create a better connection with product editing.
- Added icons to each one of the sections

| Before  | After  |
|---|---|
| <img width="1005" alt="Screenshot 2024-03-14 at 22 31 46" src="https://github.com/woocommerce/woocommerce/assets/15730971/3bfdbdf8-eb43-4ce1-957f-d3ea7024a548">  | <img width="1006" alt="Screenshot 2024-03-14 at 22 13 53" src="https://github.com/woocommerce/woocommerce/assets/15730971/90bc5b64-79cf-42da-a029-ac526e146603"> |

| Before (mobile view)  | After (mobile view)  |
|---|---|
|  <img width="406" alt="Screenshot 2024-03-14 at 22 32 01" src="https://github.com/woocommerce/woocommerce/assets/15730971/49d270f0-61a5-4286-bdae-31d1b345c9dc"> | <img width="405" alt="Screenshot 2024-03-14 at 22 14 05" src="https://github.com/woocommerce/woocommerce/assets/15730971/ec8d8eaa-b04c-4e1c-9d3b-4d599f84402a"> |





Closes #45539
Closes #45511



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure you have the WooCommerce Beta Tester plugin installed and activated
2. Under Tools > WCA Test Helper > Features, make sure customize-store is enabled
3. Under Tools > WCA Test Helper > Tools, click on **Reset Customize Your Store** and **Delete all products**
4. Head over to WooCommerce > Home > Customize your store
5. Click on "Start designing" and proceed to the Pattern Assembler
6. Click on Done
7. Make sure that you land on the following transitional page:

<img width="1006" alt="Screenshot 2024-03-14 at 22 13 53" src="https://github.com/woocommerce/woocommerce/assets/15730971/149cefa8-334f-419d-9417-b4974d11a8d4">

8. Make sure all links on the page work as expected. The new "Go to products" link should lead you to the products list `wp-admin/edit.php?post_type=product`:

<img width="1370" alt="Screenshot 2024-03-14 at 22 38 54" src="https://github.com/woocommerce/woocommerce/assets/15730971/1013db89-87ac-452b-a156-c335ac09cbbc">

9. Make sure the "Share feedback" and the "View store" buttons work as expected
10. Make sure you can visualize the following while on mobile view:

<img width="405" alt="Screenshot 2024-03-14 at 22 14 05" src="https://github.com/woocommerce/woocommerce/assets/15730971/625321aa-b37e-441d-96da-f28604dd8d61">



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Introduce the new Transitional page to the customize your store.

#### Comment 

</details>
